mlflow/spark_model/__init__.py

```diff
@@ -0,0 +1,40 @@
+import pandas
+
+from mlflow import tracking
+
+
+def load_udf(spark, path, run_id=None, result_type='double'):
```

Comment:
Why did we move this to a new module now? Seems like it should stay back in pyfunc.

Reply:
Oh, I thought you asked me to move it some time ago, I probably misunderstood. I'll move it back to pyfunc.